src/ext/java/org/opentripplanner/ext/vectortiles/layers/LayerFilters.java

```diff
@@ -0,0 +1,73 @@
+package org.opentripplanner.ext.vectortiles.layers;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.temporal.TemporalAdjusters;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.function.Supplier;
+import org.opentripplanner.apis.gtfs.model.LocalDateRange;
+import org.opentripplanner.transit.model.network.TripPattern;
+import org.opentripplanner.transit.model.site.RegularStop;
+import org.opentripplanner.transit.model.timetable.Trip;
+import org.opentripplanner.transit.service.PatternByServiceDatesFilter;
+import org.opentripplanner.transit.service.TransitService;
+
+/**
+ * Predicates for filtering elements of vector tile layers. Currently only contains predicates
+ * for {@link RegularStop}. Once more types need to be filtered, this may need some refactoring.
+ */
+public class LayerFilters {
+
+  /**
+   * No filter is applied: all stops are included in the result.
+   */
+  public static final Predicate<RegularStop> NO_FILTER = x -> true;
+
+  /**
+   * Returns a predicate which only includes stop which are visited by a pattern that is in the current
+   * TriMet service week, namely from Sunday to Sunday.
+   */
```

Comment:
I don't know if we need to specify trimet here.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/6003/commits/87991acf5bdb8ab73b898b0c0247d25a4eb726c3